doc/orchestrator.rst

```diff
@@ -165,3 +165,399 @@ A full example of configuring KeyDB to run in SmartSim is shown below
+with the ``Experiment`` object. The Experiment object is intended to be instantiated
+once and utilized throughout the workflow runtime. Begin by creating a ``Experiment``
+object and assign it to the ``exp`` variable. In this example, we instantiate the
+``Experiment`` object with the name ``getting-started-multidb``. We setup the SmartSim
+``logger`` as well for use within the Experiment. For more information on the SmartSim,
+click here.
+
+.. code-block:: python
+
+  from smartsim import Experiment
+  from smartredis import Client
+  from smartredis.log import get_logger
+  import numpy as np
+
+  exp = Experiment("getting-started-multidb", launcher="auto")
+  logger = get_logger("Multidb Experiment Log")
+
+In the context of this ``Experiment``, it's essential to create and launch
+the databases as a preliminary step before any other components. We aim
+to showcase the multi-database automation capabilities of SmartSim, so we
+create two types of Orchestrators in the workflow: a co-located Orchestrator and a
+clustered Orchestrator.
+
+To create an Orchestrator, utilize the ``create_database()`` function located
+in the ``Experiment`` class. When launching more than one Orchestrator, the
+function requires specifying a unique database identifier
+argument named ``db_identifier``. The ``db_identifier``
+serves as a unique identifier for each database being launched. Usage of the ``db_identifier``
+is show in the example below.
+
+Here's how to create the databases:
+
+For the co-located database:
+
+.. code-block:: python
+
+  colocated_orch = exp.create_database(port=6379, db_nodes=1, interface="ib0", db_identifier="single_shard_db_identifier")
+  exp.generate(colocated_orch, overwrite=True)
+
+For the clustered database:
+
+.. code-block:: python
+
+  clustered_orch = exp.create_database(port=6380, db_nodes=3, interface="ib0", db_identifier="clustered_db_identifier")
+  exp.generate(clustered_orch, overwrite=True)
+
+.. note::
+  The ``exp.generate()`` function accepts entities within the workflow, such as Model,
+  Ensemble, and Orchestrator. This function organizes the outputs of these entities
+  under the associated Experiment folder. In this example, the Experiment folder is
+  named ``getting-started-multidb/``. Within this folder, two Orchestrator subfolders will
+  be created, namely ``single_shard_db_identifier/`` and ``clustered_db_identifier/``. It's
+  important to note that the names of the Orchestrator subfolders are determined by
+  the db_identifier value we set when instantiating an ``Orchestrator``.
+
+Next, pass the database instances, ``clustered_orch`` and ``colocated_orch``, to the following command: ``exp.start()``:
+
+.. code-block:: python
+
+  exp.start(colocated_orch, clustered_orch, summary=True)
+
+The ``Experiment.start()`` function launches the ``Orchestrators`` for use within the workflow. In other words, the function
+deploys the databases on the allocated compute resources.
+
+.. note::
```

Comment:
This level of detail is not necessary here. These environment variables are largely not seen by the user